docs/eventing/sources/README.md

```diff
@@ -1,31 +1,16 @@
-<!--
-This is a generated file and should not be changed manually. All changes should follow the
-procedure:
-
-1. Update the information in [`sources.yaml`](./sources.yaml).
-
-2. Run the generator tool:
-    ```shell
-    go run docs/eventing/sources/generator/main.go
-    ```
--->
-
 Event Sources are Kubernetes Custom Resources which provide a mechanism for registering interest in
-a class of events from a particular software system. Since different event sources may be described
-by different Custom Resources, this page provides an index of the available source resource types as
-well as links to installation instructions.
-
-This is a non-exhaustive list of Event sources for Knative.
-
-
-### Inclusion in this list is not an endorsement, nor does it imply any level of support.
-
+a class of events from a particular software system. 
```

Comment:
Maybe let's not say software system. The events generally come from an event source, channel, broker, etc, right? I think we need to call out better what is an actual viable event "producer" in the scope of Knative. WDYT?

Also I'd say "registering event subscriber interest" or something that explains who or what is registering interest in the events.